packages/nuxt/README.md

```diff
@@ -62,25 +62,35 @@ export default defineNuxtConfig({
 });
 ```
 
-2. Add a `sentry.client.config.(js|ts)` file to the root of your project:
+### 3. Client-side setup
+
+Add a `sentry.client.config.(js|ts)` file to the root of your project:
 
 ```javascript
 import * as Sentry from '@sentry/nuxt';
 
 if (!import.meta.env.SSR) {
   Sentry.init({
     dsn: env.DSN,
-    replaysSessionSampleRate: 0.1,
-    replaysOnErrorSampleRate: 1.0,
   });
 }
 ```
 
-### 3. Server-side Setup
+### 4. Server-side setup
+
+Add a `sentry.server.config.(js|ts)` file to the root of your project:
+
+```javascript
+import * as Sentry from '@sentry/nuxt';
 
-todo: add server-side setup
+if (import.meta.env.SSR) {
+  Sentry.init({
+    dsn: env.DSN,
+  });
+}
```

Comment:
h: I think if we set a mode in `addPluginTemplate` we can get rid of the if statements around sentry.init.